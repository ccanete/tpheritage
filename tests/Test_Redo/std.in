R test 1 3 2 6
UNDO
LIST
REDO
LIST
EXIT