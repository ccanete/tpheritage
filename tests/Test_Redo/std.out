OK
# Rectangle test of coordinates (1, 3) and (2, 6) has been created.
OK
# The last action has been canceled.

OK
# The last action has been restored.
R test 1 3 2 6

# The program is being closed.